rcsb/workflow/wuw/BcifWorkflow.py

```diff
@@ -0,0 +1,93 @@
+##
+# File:    BcifWorkflow.py
+# Author:  James Smith
+# Date:    21-Feb-2025
+##
+
+"""
+Run workflow for pdb entries specified in input list.
+"""
+
+__docformat__ = "google en"
+__author__ = "James Smith"
+__email__ = "james.smith@rcsb.org"
+__license__ = "Apache 2.0"
+
+import os
+import logging
+from rcsb.workflow.bcif.task_functions import convertCifFilesToBcif
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler()
+formatter = logging.Formatter(
+    fmt="%(asctime)s @%(process)s [%(levelname)s]-%(module)s: %(message)s"
+)
+handler.setFormatter(formatter)
+logger.addHandler(handler)
```

Comment:
After doing the above, I think here you want to _remove_ the logging setup stuff, since that would have already been setup by the `BcifExec`. So, here, the only thing you'd keep is:

```python
logger = logging.getLogger(__name__)
```